include-build/roborazzi-core/src/jvmTest/kotlin/io/github/takahirom/roborazzi/ProcessOutputImageAndReportTest.kt

```
@@ -0,0 +1,69 @@
+      resizeScale: Double,
+      contextData: Map<String, Any>,
+      imageIoFormat: ImageIoFormat,
+    ) = error("save should not be reached")
+
+    override fun differ(
+      other: RoboCanvas,
+      resizeScale: Double,
+      imageComparator: ImageComparator
+    ): ImageComparator.ComparisonResult = error("differ should not be reached")
+
+    override fun release() {}
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)